lib/browser-auto-tracking.js

```diff
@@ -18,6 +18,12 @@ export function initAutoTrackingCore(lib) {
       shareUuidAcrossDomains: false
     }, obj);
 
+    const defaultDomElementOptions = {
+      recordTextContent: true,
+      redactTextContent: true
```

Comment:
only 1 of these is necessary. We can get rid of the feature for redaction if recording itself is configurable

Reply:
is there value in indicating that an element does in fact have text content without knowing what it is? That was the use case I was envisioning

Reply:
we're into proposing alternative features here -> 1 way hashing instead of pure redaction. Regardless I retract my "changes requested" and approve your changes instead.

Reply:
Yes, there is definitely value in that. 

Going further, dan suggested a one-way hash so that we can organize by text content even if we don't know what the content is. 

To go a little bit further, optimal for this would be:
if text content exists: 
-  take lowercase 
-  strip all newlines and whitespace 
-  generate the hash. 

Also, ideally I would also get the length of text if it's not too much trouble, i.e. a separate field that returns the length (# characters) of the stripped textcontent value.  This helps me identify when the textcontent is too long and likely contains way too much content to be useful.